doc/documentation-main.html

```diff
@@ -2923,6 +2975,18 @@ <h3><a class="section" name="yaml">YAML</a></h3>
 
 You might find the YAML file format easier to read and to maintain. YAML files are also recognized by the TestNG Eclipse plug-in. You can find more information about YAML and TestNG in this <a href="http://beust.com/weblog/2010/08/15/yaml-the-forgotten-victim-of-the-format-wars/">blog post</a>.
 
+<!------------------------------------
+  Dry Run for TestNG
+  ------------------------------------>
+
+<h3><a class="section" name="dryrun">Dry Run for your tests</a></h3>
+
+TestNG now lets you do a dry run of your tests. A dry run is essentially a mode of execution, where TestNG would only tell you what all test methods would get executed, but not actually execute them.
```

Comment:
"When launched in dry run mode, TestNG will display a list of the test methods that would be invoked but without actually calling them."

Reply:
fixed.